documentation/api/io.strimzi.api.kafka.model.AutoRestart.adoc

```diff
@@ -0,0 +1,35 @@
+Connectors and tasks that are in the FAILED state are automatically restarted.
+
+A back-off algorithm is applied to automatic restarts of each connector and it's tasks.
+When a connector is failing an auto restart attempt will happen at first and second reconcile then attempts while be delayed
+incrementing the time between automatic restarts by 2 minutes up to a maximum of 30 minutes, at which point no further
+automatic restarts are attempted, and the connector or tasks are left in the FAILED state.
```

Comment:
Should we add a sentence here to explain what the user should do next? Is the expectation that they then would add an annotation to request the connector be restarted? Once the connector/task has been successfully restarted will automatic restarts work again if it later fails?

Reply:
Yes I missed parts of what user should expect after auto restart and what should they do next. I will add them.